CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+# Ed-Fi Code of Conduct
+
+Please see [Code of Conduct](https://techdocs.ed-fi.org/x/44BmAQ) in Ed-Fi Tech
+Docs.
```

Comment:
Please [use this new one[(https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/CODE_OF_CONDUCT.md) instead. You can copy the entire file into here, not just a link to it.